libs/SalesforceSDKCore/Images.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
A couple of thoughts on this.

1. We should consider whether this asset catalog should live in shared/resources/ with the other resources we currently have in the SDK.  The catalog is going to have to be pulled into the Cordova plugin, if that changes the consideration at all.
2. We should rename the catalog to something more specific—again, because it's going to end up logically separated in apps that use static libraries, like Cordova.  Something like SalesforceSDKAssets.xcassets, or something similar?